tests/integration/test_vulnerability_detector/test_providers/test_download_feeds.py

```diff
@@ -94,14 +96,25 @@ def test_download_feeds(get_configuration, configure_environment, restart_module
     if os:
         callback_system_log = f"{system_log} {os}"
 
-    # Check that it starts the feed download
-    wazuh_log_monitor.start(timeout=VULN_DETECTOR_GLOBAL_TIMEOUT,
-                            callback=make_vuln_callback(f"Starting '{callback_system_log}' database update"),
-                            error_message=f"Could not find the provider {callback_system_log} feed download")
-
-    # Check that the feed is downloaded successfully
-    wazuh_log_monitor.start(timeout=download_timeout,
-                            callback=make_vuln_callback(f"The update of the '{callback_system_log}' feed finished \
-                                                        successfully"),
-                            error_message=f"The provider {callback_system_log} download has taken more than \
-                                            {download_timeout/60} minutes")
+    try:
+        # Check that it starts the feed download
+        wazuh_log_monitor.start(timeout=VULN_DETECTOR_GLOBAL_TIMEOUT,
+                                callback=make_vuln_callback(f"Starting '{callback_system_log}' database update"),
+                                error_message=f"Could not find the provider {callback_system_log} feed download")
+
+        # Check that the feed is downloaded successfully
+        wazuh_log_monitor.start(timeout=download_timeout,
+                                callback=make_vuln_callback(f"The update of the '{callback_system_log}' feed finished \
+                                                            successfully"),
+                                error_message=f"The provider {callback_system_log} download has taken more than \
+                                                {download_timeout/60} minutes")
+    finally:
+        control_service('stop', daemon='wazuh-db')
+
+        # Clean NVD tables when the download has finished
+        clean_vd_tables()
+
+        # Wait for cleaning NVD tables
+        time.sleep(5)
```

Comment:
Since you are only using `sleep` function from `time` module, it is better to import just this function instead of the complete module: `from time import sleep`. This reduces the overhead of importing the complete module.
```suggestion
        sleep(5)
```

Reply:
Done in 47d1f01d40d969c0b81515c131051a3e24afdc6a